nucleus/packager/nucleus-hk2/pom.xml

```diff
@@ -97,15 +97,16 @@
         </dependency>
          <dependency>
             <groupId>org.glassfish.hk2.external</groupId>
-            <artifactId>javax.inject</artifactId>
+            <artifactId>jakarta.inject</artifactId>
         </dependency>
         <dependency>
             <groupId>org.glassfish.hk2.external</groupId>
-            <artifactId>asm-repackaged</artifactId>
+            <artifactId>aopalliance-repackaged</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.glassfish.hk2.external</groupId>
-            <artifactId>aopalliance-repackaged</artifactId>
+            <groupId>fish.payara.external</groupId>
+            <artifactId>tiger-types</artifactId>
+            <version>${project.version}</version>
```

Comment:
This is not a new dependency, but previously HK2 repackaged it and included it. HK2 no longer uses it, so we have to repackage it ourselves.